src/index.js

```diff
@@ -360,6 +361,72 @@ class Compute extends common.Service {
       }
     );
   }
+  /**
+   * Create an instance template.
+   *
+   * @see [Instance Templates: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instanceTemplates/insert}
+   *
+   * @param {string} name - The name of the target image.
+   * @param {object} parameters - The settings of the template, see the
```

Comment:
I don't believe this is in the code. It looks like it's just "name" and "options".

Reply:
Fixed!